static/scripts/app.js

```diff
@@ -112,9 +91,9 @@ app._fetch = async function(provider, c = new FormData()) {
  * @return {Promise} Resolves if credential info is available.
  */
 app._autoSignIn = async function(silent) {
-  if (app.cmaEnabled) {
+  if (window.PasswordCredential) {
```

Comment:
Technically this is OK, but semantically, the intention seems to be to retrieve federated credentials as well, so maybe we should test for window.FederatedCredential too?